### PR TITLE
Fill in some history on tabs proposal

### PR DESCRIPTION
**Pull request by bkardell**
I guess I can take a crack at putting _something_ in the history part as a PR?

I mean, I'm sure people will want significant edits but at least it's more than 'put some stuff here'

### CODE REVIEW

File: research/src/pages/tabs.proposal.mdx [hunk 1]
```diff
@@ -14,7 +14,22 @@ showInMenu: false
...
+
+Because this was all true while the WHATWG took up HTML 5, and there were even markup oriented expressions of several of these components in systems like XUL, XAML and Flex,  the HTML specification briefly included an element known as [`<switch>`](https://whatwg.org/specs/web-apps/2006-01-01/#switch) but this was removed in 2006.  
+
+In 2015, several W3C participants got together to try to [propose work toward a standard solution through WICG](https://discourse.wicg.io/t/panels-and-panelsets/1184) for several of these use cases.  While a fair amount of the accessibility community was interested in pursuing this, the effort ultimately failed for a number of reasons, technical limitations at the time (no wide Shadow DOM availability and the rise of frameworks with their own lifecycles greatly increased the difficulties of a really great implementation).  While it got use and feedback, it was quite limited and after some experience, eventually dropped as a proposal.
```
Comment (gregwhitworth) [inline research/src/pages/tabs.proposal.mdx:29]: I think I'd prefer for this to reference, especially given the prior paragraph to reference that many component libraries have built tab components and... next sentence regarding developer desire

File: research/src/pages/tabs.proposal.mdx [hunk 2]
```diff
@@ -14,7 +14,22 @@ showInMenu: false
 
 ### Background <a href="#background" id="background"></a>
```
Comment (gregwhitworth) [inline research/src/pages/tabs.proposal.mdx:15]: You're the first one to have a full background on this. I had even asked in the template at one point what the reason for this section was and decided to not push too hard on it. I like this content - not sure if it should lead the spec however. I think near the end this makes more sense as I think the primary reason folks will come here is to look for implementation information but I think the additional WIKI type content is useful. If you're able to reduce it some I think that would be helpful as well but keep the spirit.

**Review by gregwhitworth — CHANGES_REQUESTED** [review]

**Comment by una** [comment]
This seems like a great place to link to a blog post or a wiki, but seems non-normative for a spec proposal. Would it be possible to move this to a post and link to that post?

**Comment by bkardell** [comment]
I'm happy to close this and let someone else take a crack at it?  It just currently has nothing, really which seems unacceptable to me.  I'm happy to do a post someone can link to with a simple "it has a history in attempts to standardize various levels of solution to these problems." containing standardization levels of history if that is helpful.

File: research/src/pages/tabs.proposal.mdx (hunk 2 above)
Comment (bkardell) [reply research/src/pages/tabs.proposal.mdx:15]: I rewrote this, much shorter, but I still don't think it is what you want.

**Comment by gregwhitworth** [comment]
> I'm happy to close this and let someone else take a crack at it? It just currently has nothing, really which seems unacceptable to me. I'm happy to do a post someone can link to with a simple "it has a history in attempts to standardize various levels of solution to these problems." containing standardization levels of history if that is helpful.

Yeah, I think I'd prefer a synopsis that links to a blog post rather than making the spec really long. We'll leave this open for now.

**Comment by gregwhitworth** [comment]
@bkardell Given all of the research that you've done and the expectation that a proposal will come in the near future, is this PR still necessary. I feel a LOT of this content is covered by your Parts research. I am personally in favor of closing this PR.

**Comment by christopherallanperry** [comment]
@gregwhitworth, I'd be happy for this PR to be closed. I have access to the button to do that now, but am not sure whether I should be the one to do that.

**Comment by andrico1234** [comment]
Closing this PR based @bkardell and @gregwhitworth 's last comments. Feel free to re-open if I've been hasty 😄

**Pull request closed without merging**